packages/react/src/transactions/adapters/profiles/UpdateFollowPolicyGateway.ts

```diff
@@ -0,0 +1,154 @@
+import {
+  omitTypename,
+  CreateSetFollowModuleTypedDataDocument,
+  CreateSetFollowModuleTypedDataData,
+  CreateSetFollowModuleTypedDataVariables,
+  SafeApolloClient,
+  FollowModuleInput,
+  CreateSetFollowModuleBroadcastItemResult,
+  RelaySuccess,
+  SetFollowModuleData,
+  SetFollowModuleVariables,
+  SetFollowModuleDocument,
+} from '@lens-protocol/api-bindings';
+import { lensHub } from '@lens-protocol/blockchain-bindings';
+import { NativeTransaction, Nonce } from '@lens-protocol/domain/entities';
+import {
+  FollowPolicyConfig,
+  FollowPolicyType,
+  UpdateFollowPolicyRequest,
+} from '@lens-protocol/domain/use-cases/profile';
+import {
+  BroadcastingError,
+  IDelegatedTransactionGateway,
+  IOnChainProtocolCallGateway,
+} from '@lens-protocol/domain/use-cases/transactions';
+import { ChainType, Data, PromiseResult, failure, success } from '@lens-protocol/shared-kernel';
+import { v4 } from 'uuid';
+
+import { UnsignedProtocolCall } from '../../../wallet/adapters/ConcreteWallet';
+import { ITransactionFactory } from '../ITransactionFactory';
+import { SelfFundedProtocolTransactionRequest } from '../SelfFundedProtocolTransactionRequest';
+import { handleRelayError } from '../relayer';
+
+export function resolveFollowModuleParams(policy: FollowPolicyConfig): FollowModuleInput {
```

Comment:
```suggestion
function resolveFollowModuleParams(policy: FollowPolicyConfig): FollowModuleInput {
```

Reply:
I'm using it in the test here, shall I just duplicate it then?
https://github.com/lens-protocol/lens-sdk/pull/610/files#diff-ffcc5cd88013e9a979a23abcbf9ab45af09041b421b5abca262ed489059629d4R36